StripeConnect/StripeConnect/Source/Internal/Webview/MessageHandlers/PageDidLoadMessageHandler.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  PageDidLoadMessageHandler.swift
+//  StripeConnect
+//
+//  Created by Chris Mays on 8/7/24.
+//
+
+import Foundation
+
+// A message that indicates components have loaded
+class PageDidLoadMessageHandler: ScriptMessageHandler<PageDidLoadMessageHandler.Message> {
+    struct Message: Codable, Equatable {
+        let pageViewId: String
```

Comment:
nits:
- Message -> Payload
- docstring
```suggestion
    struct Payload: Codable, Equatable {
        /// A unique session ID shared with web for analytics logging
        let pageViewId: String
```